application/config/fonts.php

```diff
@@ -30,16 +30,6 @@
         ),
     ),
 
-    'font-roboto' => array(
```

Comment:
Adding a comment for instruction when adding a new font ?

Reply:
The order here is needed ? Not related ?

Reply:
These font's are also shown on the dropdown

Reply:
Yes, but order is set in XML only ? Or by here too ?
Just to know :)

Reply:
These fonts are core fonts. Different than the fonts listed in theme xml

Reply:
Thank you